src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemCsv.scala

```diff
@@ -0,0 +1,51 @@
+package nl.knaw.dans.easy.stage.fileitem
+
+import java.io.InputStream
+
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.csv.CSVParser.parse
+import org.rogach.scallop.TrailingArgsOption
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConversions._
+import scala.io.Source.fromInputStream
+import scala.util.{Success, Try}
+
+object FileItemCsv {
+  val log = LoggerFactory.getLogger(getClass)
+
+  def read(in: InputStream, conf: FileItemConf): Try[Array[FileItemSettings]] = Try {
```

Comment:
Don't drag in details about command line parsing to the proper application logic.